astroquery/mast/tests/test_mast_remote.py

```diff
@@ -691,6 +691,17 @@ def test_catalogs_query_criteria(self):
         assert isinstance(result, Table)
         assert 'PSO J254.2861-04.1091' in result['objName']
 
+        result = mast.Catalogs.query_criteria(coordinates="158.47924 -7.30962",
```

Comment:
For the unit test, I'm using one of the queries that was reported to fail by one of our users. It includes the `columns` and `sort_by` arguments, both arguments were causing the `422` error observed by both users. This unit test will catch any future issues with the criteria syntax, should they arise.